### ISSUE TITLE
The target_include not working and appear some files are missing

### ISSUE THREAD

**Issue by andreagen0r** [KDAB/KDDockWidgets#307]
Hi, I compiled the project with Qt 6.3.1, everything went fine during compilation and installation. The examples run perfectly, but when trying to make a minimal project I'm not able to make it work.

+ Kubuntu 22.04
+ Kernel: 5.15.0-46
+ Qt 6.3.1
+ Branch: 2.0 

Command I used to compile
``` shell
cmake -S.. -G Ninja -DCMAKE_PREFIX_PATH=/home/MyUser/Qt/6.3.1/gcc_64 -DKDDockWidgets_QT6=ON -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=/usr/local/
ninja  
sudo ninja install  
```
Up to this point, everything went well.
So I copied some files from the examples to create a minimal standalone project

### CMakeLists.txt
```
cmake_minimum_required(VERSION 3.16)

project(MinimalDOCK VERSION 0.1 LANGUAGES CXX)

set(CMAKE_AUTOMOC ON)
set(CMAKE_CXX_STANDARD_REQUIRED ON)

find_package(Qt6 6.2 COMPONENTS Quick REQUIRED)
find_package(KDDockWidgets-qt6 CONFIG)

qt_add_executable(${PROJECT_NAME}
    main.cpp
)

qt_add_qml_module(${PROJECT_NAME}
    URI untitled
    VERSION 1.0
    QML_FILES main.qml Another.qml Guest.qml Guest1.qml Guest2.qml Guest3.qml
)

target_link_libraries(${PROJECT_NAME}
    PRIVATE Qt6::Quick KDAB::kddockwidgets)

```

### main.cpp
```
#include <kddockwidgets/Config.h>
#include <kddockwidgets/Platform_qtquick.h>
#include <kddockwidgets/kddockwidgets/ViewFactory.h>
#include <kddockwidgets/private/DockRegistry.h>
#include <kddockwidgets/views/DockWidget_qtquick.h>

#include <QGuiApplication>
#include <QQmlApplicationEngine>

int main(int argc, char *argv[]) {
  QGuiApplication app(argc, argv);

  KDDockWidgets::initFrontend(KDDockWidgets::FrontendType::QtQuick);

  auto flags = KDDockWidgets::Config::self().flags();
  KDDockWidgets::Config::self().setFlags(flags);

  QQmlApplicationEngine appEngine;
  KDDockWidgets::Platform_qtquick::instance()->setQmlEngine(&appEngine);
  appEngine.load((QUrl(u"qrc:/untitled/main.qml"_qs)));

  auto dw1 = new KDDockWidgets::Views::DockWidget_qtquick("Dock #1");

  dw1->setGuestItem(QStringLiteral("qrc:/Guest1.qml"));
  dw1->resize(QSize(800, 800));
  dw1->show();

  auto dw2 = new KDDockWidgets::Views::DockWidget_qtquick("Dock #2");
  dw2->setGuestItem(QStringLiteral("qrc:/Guest2.qml"));
  dw2->resize(QSize(800, 800));
  dw2->show();

  auto dw3 = new KDDockWidgets::Views::DockWidget_qtquick("Dock #3");
  dw3->setGuestItem(QStringLiteral("qrc:/Guest3.qml"));

  dw1->addDockWidgetToContainingWindow(dw3, KDDockWidgets::Location_OnRight);

  // Access the main area we created in QML with DockingArea {}
  auto mainArea =
      KDDockWidgets::DockRegistry::self()->mainDockingAreas().constFirst();
  mainArea->addDockWidget(dw2, KDDockWidgets::Location_OnTop);

  return app.exec();
}
```

All other qml files are just a copy from the qtquick/dockwidgets example directory

![Screenshot main.cpp error](https://snipboard.io/q1MUeu.jpg)

As you can see below, the file being searched for does not exist.  
![Filesystem](https://snipboard.io/UWGDBZ.jpg)

**Comment by iamsergio** [comment]
Fixed in 282c3106a , please pull and try again

**Comment by andreagen0r** [comment]
@iamsergio 
```
/home/tmp/MinimalDock/main.cpp:3: error: 'kddockwidgets/kddockwidgets/ViewFactory.h' file not found
/home/tmp/MinimalDock/main.cpp:3:10: fatal error: 'kddockwidgets/kddockwidgets/ViewFactory.h' file not found
#include <kddockwidgets/kddockwidgets/ViewFactory.h>
```
![Error](https://snipboard.io/izbBwI.jpg)
